### PR TITLE
[MRG+1] Added documentation about accessing header values

### LINKED ISSUE #2320: Document headers methods such as getlist()
It appears that current documentation does not mention how you can get values from HTTP `Headers` object.
Things like `response.headers.getlist('Set-Cookie')` to get all cookies are not explained (see https://github.com/scrapy/scrapy/issues/2319)

### PR DESCRIPTION
**Pull request by bopace**
Adds documentation to resolve #2320

### CODE REVIEW

File: docs/topics/request-response.rst [hunk 1]
```diff
@@ -507,7 +507,13 @@ Response objects
 
     .. attribute:: Response.headers
 
-        A dictionary-like object which contains the response headers.
+        A dictionary-like object which contains the response headers. Values can
+        be accessed from the :class:`headers` object by using :meth:`get` to
```
Comment (redapple) [inline docs/topics/request-response.rst]: I don't think _"from the :class:`headers` object by"_ is necessary

**Comment by redapple** [comment]
Thanks @bopace ! I nitpicked a little.

File: docs/topics/request-response.rst (hunk 1 above)
Comment (eliasdorneles) [inline docs/topics/request-response.rst]: yup, that's not resolving to a proper link -- no need to use special markup here, just headers is good.

Comment (bopace) [inline docs/topics/request-response.rst]: Thanks for the heads up. I committed a fix just now.

**Comment by bopace** [comment]
Thanks! How does that change look?

**Comment by eliasdorneles** [comment]
Looks good, thanks @bopace !

**Pull request merged**